apps/opik-frontend/src/components/pages-shared/automations/ProjectsSelectBox.tsx

```diff
@@ -93,6 +93,7 @@ interface BaseProjectsSelectBoxProps {
   disabled?: boolean;
   minWidth?: number;
   customOptions?: DropdownOption<string>[];
+  align?: "start" | "end" | "center";
```

Comment:
i would use a type from CSSProperties 
```suggestion
  align: CSSProperties["alignItems"];
```